less/yeti.less

```diff
@@ -0,0 +1,524 @@
+#yeti {
```

Comment:
On the less, there is a lot of work that could be done to homogenize styling, e.g. variables for sizes and colors, but this is a work that is also needed for the whole site > let's leave that appart for now